### DISCUSSION TITLE
StaticProblem ('cruise') Warning

### DISCUSSION THREAD

**Discussion by erkhans** [mdolab/dafoam#614] [General]
Hello, I am trying to run a 2-way dynamic FSI for a wind turbine.  I am getting the warning shown in the figure below. Then, cruise coupling fails to converge. What might be the cause of this warning? How it can be resolved? Could you please help me with it? Thank you.
![e1](https://github.com/mdolab/dafoam/assets/114105732/e435aa26-7692-4047-90a9-9e6dbe0df83e)

**Comment by friedenhe** [comment]
It seems that the nonlinear FSI converged for 5 orders of magnitude and stalled... Not sure what caused it though.. I suggest you increase the stiffness of your structure and see if it converges.

**Reply by erkhans** [reply]
Hello, I've changed the mesh. Now, the StaticProblem ('cruise') warning has disappeared, but it still does not converge. Do you have any suggestions for it? Thank you.
![er](https://github.com/mdolab/dafoam/assets/114105732/3045233a-176d-4c9b-97d1-56d990f4af5e)

**Comment by friedenhe** [comment]
Please set a loose tolerance.. The NLBGS has already converged for almost 7 orders.
